_people/aydin_saribudak.md

```diff
@@ -0,0 +1,23 @@
+---
+name: "Aydin Saribudak"
+collection: people
+permalink: /people/aydins
+position: phd
+title: "PhD Student"
+joined: 2019
+avatar: aydins.jpg
+contacts:
+  email: "aydin.saribudak@rutgers.edu"
+  office: "CoRE 707"
+  [scholar][scholarlink] 
+  github: https://github.com/aydinsaribudak
+  linkedin: https://www.linkedin.com/in/aydin-saribudak-1113b927
+  researchgate: https://www.researchgate.net/profile/Aydin_Saribudak
+[scholar]: https://scholar.google.com/citations?user=Cx_5JNoAAAAJ&hl=en&oi=ao
```

Comment:
This should be inside the contacts part of the page. See above comment.

Reply:
I update the file as requested. I make the "scholar' inside the contacts.